Tests/configure_and_test_integration_instances.py

```diff
@@ -85,6 +84,10 @@ class Running(IntEnum):
     WITH_LOCAL_SERVER = 2
 
 
+def generate_user_agent(build_number):
```

Comment:
Just a suggestion. What's wrong with that name?
```suggestion
def get_custom_user_agent(build_number):
```